ports/esp32/boards/sdkconfig.base

```diff
@@ -63,6 +63,16 @@ CONFIG_MBEDTLS_ASYMMETRIC_CONTENT_LEN=y
 # Disable ALPN support as it's not implemented in MicroPython
 CONFIG_MBEDTLS_SSL_ALPN=n
 
+# Disable slow or unused EC curves
+CONFIG_MBEDTLS_ECP_DP_BP256R1_ENABLED=n
+CONFIG_MBEDTLS_ECP_DP_BP384R1_ENABLED=n
+CONFIG_MBEDTLS_ECP_DP_BP512R1_ENABLED=n
+CONFIG_MBEDTLS_ECP_DP_CURVE25519_ENABLED=n
```

Comment:
In what way are these unused?  Do you mean that the other ports don't enable them by default?  Or that they can never be utilised during a connection?

Reply:
Ah, I see, they were removed from other ports in 68f166dae9ad6dfd94038d5f4394defbb44238af